lib/Cake/Test/Case/Console/Command/Task/ExtractTaskTest.php

```diff
@@ -469,10 +469,10 @@ public function testExtractModelValidationInPlugin() {
 		$this->Task->execute();
 		$result = file_get_contents($this->path . DS . 'test_plugin.pot');
 
-		$pattern = preg_quote('#Model/TestPluginPost.php:validation for field title#', '\\');
+		$pattern = preg_quote('#Model/TestPluginPost.php:validation for field title#', '/');
 		$this->assertRegExp($pattern, $result);
 
-		$pattern = preg_quote('#Model/TestPluginPost.php:validation for field body#', '\\');
+		$pattern = preg_quote('#Model/TestPluginPost.php:validation for field body#', '/');
```

Comment:
Remove `preg_quote` at all, it left from old versions when generated `.po` contained `\` on Windows.